### ISSUE TITLE
Cannot log into the Dashboard, URL opens a random Album Art

### ISSUE THREAD

**Issue by renatopinto89** [AzuraCast/AzuraCast#7091]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Stable Channel

### Current AzuraCast Version

v0.19.7 Stable • Docker • PHP 8.3

### What happened?

Hhello everyone,

When I try to log into my Azuracast Instalation, istead of the login splash page, I get one random album art (https://prnt.sc/YGzdni-r-Z8I).
The only way to get the login page, is to access it through the direct IP address of the Server
This already happened a couple of times before, but restarting the AzuraCast services would fix it, but now it goes back to this state after a couple minutes since restart.
Restarting the NGINX Service through the "Administration" page, throws an "An error occurred and your request could not be completed." message

Any tips?

Thank you so much in advance

### Relevant log output

```shell
NGINX Logs

2024/05/08 11:01:16 [error] 242#242: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 2.80.200.243, server: localhost, request: "GET /api/admin/log/service_nginx?position=7872 HTTP/2.0", upstream: "fastcgi://127.0.0.1:6090", host: "login-na2.lightmanstreams.com", referrer: "https://login-na2.lightmanstreams.com/admin/logs"
2024/05/08 11:01:16 [error] 242#242: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 2.80.200.243, server: localhost, request: "GET /api/admin/log/service_nginx?position=7872 HTTP/2.0", upstream: "fastcgi://127.0.0.1:6090", host: "login-na2.lightmanstreams.com", referrer: "https://login-na2.lightmanstreams.com/admin/logs"
2024/05/08 11:01:19 [error] 242#242: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 2.80.200.243, server: localhost, request: "GET /api/admin/log/service_nginx?position=7872 HTTP/2.0", upstream: "fastcgi://127.0.0.1:6090", host: "login-na2.lightmanstreams.com", referrer: "https://login-na2.lightmanstreams.com/admin/logs"
2024/05/08 11:01:27 [error] 242#242: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 2.80.200.243, server: localhost, request: "GET / HTTP/2.0", upstream: "fastcgi://127.0.0.1:6090", host: "login-na2.lightmanstreams.com"
```

**Comment by BusterNeece** [comment]
@renatopinto89 That's a _very_ unusual error to be having. It implies that something upstream of AzuraCast (maybe a reverse proxy or something similar) is aggressively misconfigured.

Is there _anything_ between the AzuraCast installation and the web in this case?

**Comment by renatopinto89** [comment]
Hi @BusterNeece 

Thank you so much for your reply.

Nothing else is running in that Server, only the AzuraCast docker container.

DNS is managed through Cloudflare, but as DNS Only, not cached

Kind Regards

**Comment by BusterNeece** [comment]
@renatopinto89 Can you try switching to the Rolling Release channel and updating to the latest version? We have a working theory of what might be causing this and want to see if our changes resolve anything.

**Comment by renatopinto89** [comment]
@BusterNeece Hi Buster, thank you so much for the reply. 

I did switch to rolling release and now monitoring status.

Something I might add, last night, the Front-End completely crashed again and saw this in the logs

azuracast  | {"level":"error","ts":1715396738069158575,"logger":"server","msg":"no free workers in the pool, wait timeout exceed","reason":"no free workers","internal_event_name":"EventNoFreeWorkers","error":"worker_watcher_get_free_worker: NoFreeWorkers:\n\tcontext deadline exceeded"}
azuracast  | {"level":"error","ts":1715396738069212025,"logger":"http","msg":"execute","start":1715396678068023353,"elapsed":60.001187932,"error":"static_pool_exec: NoFreeWorkers:\n\tstatic_pool_exec:\n\tworker_watcher_get_free_worker:\n\tcontext deadline exceeded"}

**Comment by BusterNeece** [comment]
@renatopinto89 I'm assuming that this issue is fixed now as of the latest stable (0.20.0) or the latest Rolling Release?

**Comment by renatopinto89** [comment]
Hi Buster
Everything seems to be running normally without those crashes since the latest stable (0.20.0).
Thank you so much for the follow up and for your help.

Kind Regards
